TWLight/graphs/views.py

```diff
@@ -345,3 +378,20 @@ def _write_data(self, response):
 
         row = [path.encode('utf-8'), path_count]
         writer.writerow(row)
+
+class CSVUserLanguage(_CSVDownloadView):
+    def _write_data(self, response):
+        csv_queryset = UserProfile.objects.all()
+
+        data = get_user_language_data(
+            csv_queryset,
+            data_format=PYTHON
+        )
+
+        writer = csv.DictWriter(response, fieldnames=['label', 'data'])
+
+        writer.writerow({'label': _('Language'),
```

Comment:
It might not be a bad idea to do a
```
.encode('utf-8')
```
on the language to prevent the server errors we have seen on some of the downloads occasionally. That's what I ended up doing with the paths.

Reply:
`get_user_language_data` already uses force_unicode on language (graphs/helpers.py L134).

Reply:
duh